osu.Game/Beatmaps/Formats/OsuLegacyDecoder.cs

```diff
@@ -56,36 +59,37 @@ private enum Section
             TimingPoints,
             Colours,
             HitObjects,
+            Variables,
         }
 
-        private void handleGeneral(Beatmap beatmap, string key, string val)

```

Comment:
`val` was used here for a reason (generally we avoid using `value` as it is a keyword in certain contexts)